docs/tutorials/generate-custom-test-data.rst

```diff
@@ -0,0 +1,120 @@
+.. _ksql_generate-custom-test-data:
+
+Generate custom test data by using the ksql-datagen tool

```

Comment:
suggest using title case, e.g. `Generate Custom Test Data by Using the ksql-datagen Tool`